probe/kubernetes/reporter.go

```diff
@@ -168,6 +168,13 @@ var (
 			Rank:  1,
 		},
 	}
+
+	DescribeControl = report.Control{
+		ID:    Describe,
+		Human: "Describe",
+		Icon:  "fa fa-file-text",
+		Rank:  2,
+	}
```

Comment:
![image](https://user-images.githubusercontent.com/1216874/55959893-0284ed00-5c6c-11e9-9855-f50db02fd033.png)

It would be nice if _Describe_ and _Remove_ switched places in Pod details, perhaps by bumping the _DeletePod_ rank to value 3.

Reply:
Sure @fbarl 
Thanks for the review.